### ISSUE TITLE
Wavebox 3.2.0-1 Hangs on Ubuntu 14.04

### ISSUE THREAD

**Issue by sabhain** [wavebox/waveboxapp#346]
* Which version of Wavebox are you using? **[e.g. 3.2.0-1]**

* Which Operating System are you using? **[Ubuntu 14.04]**

Since the upgrade last week, Wavebox does not resume after being inactive for more than an hour or so.  It resumes just fine if the screen saver has only been on for a few minutes, but I think after 3 hours, which coincides with the duration at which I allow some accounts within Wavebox to go dormant, the app is no longer responsive.

This behavior certainly began after the recent upgrade (which I performed directly through Ubuntu Software Center).

Awesome program anyway .. it's really streamlined some things for us.

**Comment by Thomas101** [comment]
Hi, thanks for raising an issue. Judging by what you've said you have some accounts setup to sleep? If you reduce the duration of the sleep does this make the crash happen sooner?

When it does freeze, does Wavebox still respond to some of the Window controls (e.g. minimize, close etc). If you click on the tray icon does the menu here still appear and are you able to hide and show the main window?

**Comment by sabhain** [comment]
Hi,

I've turned off the sleep, and it still happens.  It appears to be more like a 5 or 6 hour issue, I think.

Wavebox does still respond to things like Minimize, close .. etc.  In fact, it still will respond to scrolling within the window.  Menus still are active, but it only responds to things that call to system activities, not items within Wavebox.  

For example, I can get to GitHub from the Wavebox menu, but I can't switch accounts or add an account, or even Quit.  It appears that anything inside Wavebox stops responding, but it's interface with the OS is still alive and functioning.

FYI, I'm using gnome-shell if that matters.

Thanks for your help.

**Comment by Thomas101** [comment]
Hi, thanks for the extra info. Do you know if you're using 32 or 64 bit Ubuntu and which install method you used to install Wavebox (deb/snap/tar/repo)?

Would you be able to give the latest beta a try? This has a newer version of electron with some fixes for sporadic crashing. There's some instructions on how to switch channel [here](https://wavebox.io/kb/beta-channel). Or if you just want to try it, downloading the tar from the [releases page](https://github.com/wavebox/waveboxapp/releases/tag/v3.2.3), quitting Wavebox and then running `./Wavebox` should work.

**Comment by sabhain** [comment]
Hi,

I'm most certainly running 64 bit.  I believe I installed via the deb file, and I have to go through the software center to upgrade it (not my best decision I don't think).  I think I should probably switch to repository .. but I tend to not update things too frequently.  I learned a long time ago to get it stable and leave it alone.

I will certainly try to give the beta a try .. but it might not be until early next week.  I'll give it a try and report back.  It's a really fantastic program .. eager to try and help where I can as a tester.

Dan
